Source/WebKit/UIProcess/ios/WKScrollView.mm

```diff
@@ -548,6 +555,24 @@ - (CGPoint)_puic_contentOffsetForCrownInputSequencerOffset:(double)sequencerOffs
 
 #endif // HAVE(PEPPER_UI_CORE)
 
+#if ENABLE(OVERLAY_REGIONS_IN_EVENT_REGION)
+- (NSArray<NSData *> *)overlayRegions
+{
+    NSMutableArray<NSData *> *overlayRegions = [NSMutableArray array];
+    for (const auto& [layerID, rect] : _overlayRegionsWithIDs) {
+        if (!CGRectIsEmpty(rect))
```

Comment:
I don't think we ever add empty rectangles, so we could probably remove the above line